docs/dev/tools/cli.md

```diff
@@ -1247,6 +1267,8 @@ viam machines part restart --machine=123 --part=456
 | `--machine-id` | The machine to create an API key for | `api-key` | **Required** |
 | `--name` | The optional name of the API key | `api-key` | Optional |
 | `--org-id` | The optional organization ID to attach the key to | `api-key` | Optional |
+| `--recursive`, `-r` | Preserve modification times and file mode bits from the source files. Default: `false`. | `part cp` | Optional |
```

Comment:
It looks like these two descriptions are swapped

Reply:
good spot